index.html

```diff
@@ -5644,3 +5644,35 @@ <h2>A project for first-time contributions</h2>
     <script type="text/javascript" src="assets/script.js"></script>
   </body>
 </html>
+
+<!-- ________ Akhik's card START ________  -->
```

Comment:
Looks like you added your card at the end of the file, you needed to paste it below the template provided in the same file near the top, for detailed instructions you can refer to step 5 in [this](https://github.com/Syknapse/Contribute-To-This-Project). Once the changes are done, I will merge your PR after the review.